flyctl/installing.html.md

```diff
@@ -27,6 +27,7 @@ If not, you can run the install script:
 ```cmd
 curl -L https://fly.io/install.sh | sh
 ```
+After that, you should add the flyctl directory to your shell rc file. Check the output of the install script for the exact command. Copy, paste and run it and the `flyctl` keyword will be available everywhere. Or for maximum efficiency, you can use the `fly` keyword! 
```

Comment:
I think we usually refer to `fly`/`flyctl` as a "command" instead of "keyword", although I see what you mean here since it's related to the your shell  knowing the word is a command.

Reply:
I agree that we use it as a command and should name it a command instead of a keyword, I've changed it to always refer to it as a command